app/build.gradle

```diff
@@ -24,8 +24,9 @@ dependencies {
     androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    implementation 'com.android.support:appcompat-v7:25.3.1'
+    implementation 'com.android.support:appcompat-v7:25.4.0'
     testImplementation 'junit:junit:4.12'
     implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    implementation 'com.android.support:design:25.3.1'
+    implementation 'com.android.support:design:25.4.0'
+    compile 'com.squareup.okhttp3:okhttp:3.8.0'
```

Comment:
OKHttp is good for now, but depending upon the use case, we might want to shift to retrofit later on.